### DISCUSSION TITLE
Cascading DropDown

### DISCUSSION THREAD

**Discussion by Leo831-ux** [Budibase/budibase#14452] [Help] [answered]
Hi

I imported the Cascading Dropdown app and followed the instructions from discussion [#14373](https://github.com/Budibase/budibase/discussions/14373) to get it working.  However, it is still not working for me.  No, I am wondering whether I have misread / misunderstood and made a mistake in the process.

New data provider as a sibling
![dataprovider](https://github.com/user-attachments/assets/f5af1228-63a9-42f1-b76e-8dafb58d71f3)

Filter settings
![binding](https://github.com/user-attachments/assets/06078cef-d4e0-4ef0-9d5b-263d4197911f)
![binding01](https://github.com/user-attachments/assets/bdf20eac-f519-42b4-8d39-530e2fbb4bfd)

**Comment by zimart** [comment]
it doesn't work for me either...

**Comment by mikesealey** [comment]
@Leo831-ux @zimart The issue initially reported in 13893 still exists, but the workaround is still valid. Are you both having issues with the workaround?

**Reply by Leo831-ux** [reply]
Hi.

Yes, I am having an issue with the workaround. So, I am not sure whether I am making a mistake somewhere when doing the workaround.

**Comment by mikesealey** [comment, marked as answer]
[14373 copy-export-1724942488167.tar.gz](https://github.com/user-attachments/files/16800861/14373.copy-export-1724942488167.tar.gz)

@Leo831-ux @zimart Here's an app-export using Data Providers to fetch Countries and Cities, where Cities filter by Country selected. 

If you need guidance on importing an app-export, head here: https://docs.budibase.com/docs/export-and-import-apps#export-an-application

**Comment by Leo831-ux** [comment]
@mikesealey

Thank you very much, it is working awesome.

One thing to note.  I had to go to the City Data Provider's Filtering option and set the filter option to "country_id."

![filteroption](https://github.com/user-attachments/assets/231950ed-07e8-4512-aacd-e141bfc8772d)


It was blank for me when importing the app.  However, it works like a dream after setting the option.

Thank you once again for the help. Much appreciated.

**Comment by Leo831-ux** [comment]
Hi

App works perfectly. It was internet issues that prevented the correct import of the app.  I tested again today in the office with no problems.
